tools/cli/commands/utils.py

```diff
@@ -68,8 +68,9 @@ class InvalidInstanceException(Exception):
 
     _MESSAGE = (
         'The specified instance, {}, does not appear '
-        'to have been created by the `datalab` tool, and '
-        'so cannot be managed by it.')
+        'to have been created by the `datalab` tool, or '
+        'from any GCE Deeplearning images. Therefore it '
+        'so cannot be managed by `datalab` tool.')
```

Comment:
The "so " here is redundant.

Reply:
Done. Can you grant me the write access to the repo? Thanks!